core/ast_node.js

```diff
@@ -474,6 +471,31 @@ Blockly.ASTNode.prototype.getOutAstNodeForBlock_ = function(block) {
   }
 };
 
+/**
+ * Find the first editable field or input with a connection on a given block.
+ * @param {Blockly.BlockSvg} block The source block of the current location.
```

Comment:
block can't be null (block.inputList would fail), so this should be {!Blockly.BlockSvg}

Reply:
Fixed